### DISCUSSION TITLE
Is it possible to disable the "seconds since last response" message on SSH Domain connections?

### DISCUSSION THREAD

**Discussion by ianliu** [wezterm/wezterm#2542] [Q&A] [answered]
I feel those messages distracting. Maybe add an option to adjust the threshold for which the message shows? I'm getting flashes of those messages while typing, and I feel like a delay of 5-7 seconds is enough to trigger the message.

The following video shows what I mean. I'm waiting for 7 seconds until I type an <kbd>l</kbd> character, which triggers a flash on the upper-right corner of the terminal:

[wezterm-flashing.webm](https://user-images.githubusercontent.com/30196/191754898-f411c666-645c-40bb-86c9-02d1e56da00f.webm)

**Comment by wez** [comment, marked as answer]
I just pushed a commit to `main` disable it by default, as I'd like to move it to the status bar.  You can use https://wezfurlong.org/wezterm/config/lua/pane/get_metadata.html#since_last_response_ms to control displaying that information.
You may use `overlay_lag_indicator = true` in the domain config to restore the old behavior.

It typically takes about an hour before commits are available as nightly builds for all platforms.  Linux builds are the fastest to build and are often available within about 20 minutes.  Windows and macOS builds take a bit longer.

Please take a few moments to try out the fix and let me know how that works out. You can find the nightly downloads for your system in the [wezterm installation docs](https://wezfurlong.org/wezterm/installation.html).

If you prefer to use packages provided by your distribution or package manager of choice and don't want to replace that with a nightly download, keep in mind that you can download portable packages (eg: a `.dmg` file on macOS, a `.zip` file on Windows and an `.AppImage` file on Linux) that can be run without permanently installing or replacing an existing package, and can then simply be deleted once you no longer need them. 

If you are eager and can [build from source](https://wezfurlong.org/wezterm/install/source.html) then you may be able to try this out more quickly.

**Reply by ianliu** [reply]
Nice, thanks!
